ShaderTool/Command/Map.cs

```diff
@@ -202,6 +198,60 @@ class Map {
                 resourceStream.Write(textureData);
             }
 
+            // Write the data names into the ressource file
+            foreach (string actorName in mapData.actorNames) {
+                string actorFilePath = Program.ResourcesFolder + @"\" + actorName + "_Actor.json";
+                if (!File.Exists(actorFilePath)) {
+                    Console.WriteLine("{0} is not a valid actor!", actorName);
+                    return WRONG_PARAMS;
+                }
+
+                ActorData actorData = JsonConvert.DeserializeObject<ActorData>(File.ReadAllText(actorFilePath));
+                // Write the local transform as a 4x4 matrix matrix into the file
```

Comment:
```suggestion
                // Write the local transform as a 4x4 matrix into the file
```
The word matrix exists two times